src/components/DetailsMetrics/ApplicationMetricCard.js

```diff
@@ -90,7 +90,7 @@ const ApplicationMetricCard = ({ metric }) => {
           </Tooltip>
         )}
       </StatsCard.Header>
-      <div className="metrics__card-body">
+      <div className="metrics__card-body metrics__card-metric">
```

Comment:
move the new class to the `metrics__card`

Reply:
@mavdryk moved metrics__card-metric class to metrics__card level